build-deploy/common/models/deployment.js

```diff
@@ -21,35 +21,43 @@ module.exports = function(Deployment) {
 
     debug('deploy to %s: %j', baseURL, deployment);
 
-    resize();
-
-    function deploy(err){
-      if (err) return cb(err);
+    deploy();
 
+    function deploy(){
       if(deployment.type === 'local') {
-        // args are: baseUrl, localdir, config, callback
-        performLocalDeployment(baseURL, cwd, 'process-manager', done);
+        performLocalDeployment(
```

Comment:
so the signature of these methods has been changed?  Where are the changes to the methods themselves?  is that a different repo/PR?